clients/imodelhub/config/rush-project.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush-project.schema.json",
+  "operationSettings": [
+    { "operationName": "build", "outputFolderNames": ["lib"] },
```

Comment:
This one can be deleted, as well as the associated changelogs. @aruniverse removed the package last week.

Reply:
done